Project.toml

```
@@ -27,12 +27,12 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 BenchmarkProfiles = "0.2.0, 0.3.0, 0.4"
 BenchmarkTools = "^0.4.2, 0.5, 0.6, 0.7, 1"
 ColorSchemes = "^3.9"
-DataFrames = "^0.21, 1.0"
+DataFrames = "^0.21, 1.0, 1.2, 1.3"
```

```suggestion
DataFrames = "^0.21, 1"
```
Does it break for 1.1? Otherwise, I believe using `1` is enough for all `1.x`.

---

It didn't break for 1.1 so it seems a good idea indeed